MAUI/Rating/Getting-Started.md

```diff
@@ -0,0 +1,189 @@
+---
+
+layout: post
+title: Getting Started with .NET MAUI Rating control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Rating (SfRating) control, its elements and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+
+# Getting Started with .NET MAUI Rating (SfRating)
+
+
+This section explains how to configure a [`SfRating`] control in a real-time scenario and also provides a walk-through on some of the customization features available in [`SfRating`] control.
+
+## Adding a .NET MAUI Rating reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI Rating] to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Inputs](https://www.nuget.org/packages/Syncfusion.Maui.Inputs) and then install it.
+
+## Handler registration 
+
+In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# hl_lines="6 7 18 19" %}
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+using Syncfusion.Maui.ListView.Hosting;
+
+namespace RatingSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureSyncfusionListView()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}   
+
+{% endhighlight %} 
+
+## Adding the .NET MAUI Rating control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample:
```

Comment:
Step 2: Add the namespace, as shown in the following code sample.

Reply:
Changed